sitewit/services.py

```diff
@@ -35,17 +35,24 @@ def _get_account_auth_header(self, account_token):
         return self._compose_auth_header((
             self._partner_id, self._partner_token, account_token))
 
-    def _get_partner_auth_header(self, subpartner_id=None):
+    def _get_partner_auth_headers(self, subpartner_id=None, remote_id=None):
```

Comment:
What do you think about naming it `remote_subpartner_id` to be consistent with other params